nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -589,6 +589,8 @@ The module update takes care of the new config syntax and the data itself (user
 
 - `python3.pkgs.flitBuildHook` has been removed. Use `flit-core` and `format = "pyproject"` instead.
 
+- `magma` now defaults to `magma-hip` instead of `magma-cuda`
```

Comment:
This is probably a good place to advertise `nixpkgs.config.cudaSupport` again.

Reply:
While you're at it, we should advertise ``nixpkgs.config.rocmSupport`` as well.
Only way to even know it exists currently is if you're looking at the ``nixpkgs`` code.

Reply:
Yeah, I was wondering if we wanted to keep it on the DL, because a refactor of the options was planned and already pretty concrete.

Reply:
Yeah it would be nice to keep it on the DL, but the reality is, it's already in nixpkgs and is thus part of the API.
It will have to go through the same deprecation process that ``cudaSupport`` will have to.